test/rest_api/test_compaction_manager.py

```diff
@@ -15,6 +15,69 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
 
+import pytest
+import sys
+import requests
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import new_test_table, unique_name
+
+# "keyspace" function: Creates and returns a temporary keyspace to be
+# used in tests that need a keyspace. The keyspace is created with RF=1,
+def new_keyspace(cql, this_dc):
+    name = unique_name()
+    cql.execute(f"CREATE KEYSPACE {name} WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 1 }}")
+    return name
```

Comment:
The more "proper" way to do this would be to use a "contextmanager" that willl create a new keyspace and return its name - but make sure the keyspace is deleted when the test ends - regardless of whether the test failed or succeeded. You can see an example of `new_test_table()` in test/cql-pytest/util.py.
But I'll let this slide and approve this PR. Unlike Alternator tests where leaving an undeleted table on AWS is bad, here the result of forgetting to delete a table is less disasterous. But it's still good practice to always think about this.